inc/search.php

```diff
@@ -178,6 +178,9 @@ function search_media(&$data,$base,$file,$type,$lvl,$opts){
     if(preg_match("/\.(jpe?g|gif|png)$/",$file)){
         $info['isimg'] = true;
         $info['meta']  = new JpegMeta($base.'/'.$file);
+    }else if(preg_match("/\.svg$/",$file)){
+        $info['isimg'] = true;
+        $info['meta']  = new JpegMeta($base.'/'.$file); //(object) array() ;
```

Comment:
Calling JpegMeta on SVG is weird at least (but not much less weird than calling it for PNG and GIF)